### DISCUSSION TITLE
Self-contained Linux (and Windows) builds

### DISCUSSION THREAD

**Discussion by Daniel15** [rogerfar/rdt-client#626] [Ideas]
It would be useful to provide [self-contained](https://learn.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained) builds so that users that don't want to use Docker don't have to install .NET manually. A self-contained app contains all the needed parts of .NET, plus all the dependencies for the app, in a single executable file. This would let you remove the "Install .NET" and "Make sure you have the .NET 9 Runtime installed" steps from the readme.

**Comment by rogerfar** [comment]
Shouldn't be too hard to add as a separate release, because it does make your release files quite a bit larger.
